src/playlist-maker.js

```diff
@@ -112,7 +101,7 @@ const indexInSources = (arr, src) => {
  * @return {Function}
  *         Returns the playlist function specific to the given player.
  */
-const factory = (player, initialList, initialIndex = 0) => {
+export default function factory(player, initialList, initialIndex = 0) {
```

Comment:
Rollup 0.50 was complaining about this because `this` was undefined - it tries to more accurately mimic how native ES6 modules will work.